src/test/java/com/endava/endabank/utils/TestUtils.java

```diff
@@ -0,0 +1,197 @@
+    }
+
+    public static UserRegisterDto getUserRegisterDto() {
+        return UserRegisterDto.builder().
+                email("user@endava.com").
+                phoneNumber("3210000000").
+                identifier("1001000000").
+                firstName("Endava").
+                lastName("User").
+                password("$2a$10$bUcuzJbChZheTqPERIqk3u7COWhAF1CV6OU.LUkCG6iZYRhXydRqW").
+                typeIdentifierId(1).build();
+    }
+
+    public static UserRegisterGetDto getUserRegisterGetDto() {
+        return new ModelMapper().
+                map(getUserNotAdmin(), UserRegisterGetDto.class);
+    }
+
+    public static UserDetailsDto userDetailsGetDto(UserPrincipalSecurity user) {
+        return new ModelMapper().
+                map(user, UserDetailsDto.class);
+    }
+
+    public static UserPrincipalSecurity getUserPrincipalSecurity() {
+        return UserPrincipalSecurity.builder().
+                id(1).
+                email("user@endava.com").
+                phoneNumber("3210000000").
+                firstName("principal").
+                isApproved(true).build();
+    }
+
+    public static UserAuthentication getUserAuthentication() {
+        Collection<SimpleGrantedAuthority> authorities = new ArrayList<>();
+        authorities.add(new SimpleGrantedAuthority("ROLE_USER"));
+        return new UserAuthentication("user@endava.com", "test-password",
+                authorities, 1,
+                true, true);
+    }
+
+    public static UpdatePasswordDto getUpdatePasswordDto() {
+        return UpdatePasswordDto.builder().
+                oldPassword("A1234567!a").
+                password("A1234567!aa").
+                rePassword("A1234567!aa").build();
+    }
+
+    public static ForgotUserPasswordToken getForgotUserPasswordToken(String token) {
+        ForgotUserPasswordToken forgotUserPasswordToken = new ForgotUserPasswordToken();
+        forgotUserPasswordToken.setUser(TestUtils.getUserNotAdmin());
+        forgotUserPasswordToken.setId(1);
+        forgotUserPasswordToken.setToken(token);
+        return forgotUserPasswordToken;
+    }
+
+    public static Map<String, Object> getSuccesfullEmailResponse() {
+        Map<String, Object> map = new HashMap<>();
+        map.put(Strings.MESSAGE_RESPONSE, Strings.MAIL_SENT);
+        map.put(Strings.STATUS_CODE_RESPONSE, HttpStatus.valueOf(HttpStatus.ACCEPTED.value()));
+        return map;
+    }
+
+    public static Map<String, Object> getFailEmailResponse() {
+        Map<String, Object> map = new HashMap<>();
+        map.put(Strings.MESSAGE_RESPONSE, Strings.MAIL_FAIL);
+        map.put(Strings.STATUS_CODE_RESPONSE, HttpStatus.valueOf(HttpStatus.SERVICE_UNAVAILABLE.value()));
+        return map;
+    }
+
+    public static UserPrincipalSecurity getUserAdminPrincipalSecurity() {
+
+        return UserPrincipalSecurity.builder().
+                id(1).
+                email("admin@endava.com").
```

Comment:
Hi,

This repository is public, so please do not use Endava information or sensitive information.